opensrp-chw/src/miecd/java/org/smartregister/chw/interactor/ChildHomeVisitInteractorFlv.java

```diff
@@ -0,0 +1,270 @@
+
+                StringBuilder stringBuilder = new StringBuilder();
+                if (famllin1m5yr.equalsIgnoreCase(context.getString(R.string.no))) {
+                    stringBuilder.append(MessageFormat.format("{0}: {1}\n", context.getString(R.string.uses_net), StringUtils.capitalize(famllin1m5yr.trim().toLowerCase())));
+                } else {
+                    stringBuilder.append(MessageFormat.format("{0}: {1} · ", context.getString(R.string.uses_net), StringUtils.capitalize(famllin1m5yr.trim().toLowerCase())));
+                    stringBuilder.append(MessageFormat.format("{0}: {1} · ", context.getString(R.string.slept_under_net), StringUtils.capitalize(llin2days1m5yr.trim().toLowerCase())));
+                    stringBuilder.append(MessageFormat.format("{0}: {1}", context.getString(R.string.net_condition), StringUtils.capitalize(llinCondition1m5yr.trim().toLowerCase())));
+                }
+                return stringBuilder.toString();
+            }
+
+            public String getYesNoTranslation(String subtitleText) {
+                if ("Yes".equals(subtitleText)) {
+                    return context.getString(R.string.yes);
+                } else if ("No".equals(subtitleText)) {
+                    return context.getString(R.string.no);
+                } else {
+                    return subtitleText;
+                }
+            }
+
+            @Override
+            public BaseAncHomeVisitAction.Status evaluateStatusOnPayload() {
+                if (StringUtils.isBlank(famllin1m5yr)) {
+                    return BaseAncHomeVisitAction.Status.PENDING;
+                }
+
+                if (famllin1m5yr.equalsIgnoreCase("Yes") && llin2days1m5yr.equalsIgnoreCase("Yes") && llinCondition1m5yr.equalsIgnoreCase("Okay")) {
+                    return BaseAncHomeVisitAction.Status.COMPLETED;
+                } else {
+                    return BaseAncHomeVisitAction.Status.PARTIALLY_COMPLETED;
+                }
+            }
+        };
+
+        BaseAncHomeVisitAction action = new BaseAncHomeVisitAction.Builder(context, context.getString(R.string.pnc_malaria_prevention))
+                .withOptional(false)
+                .withDetails(details)
+                .withFormName(Constants.JSON_FORM.CHILD_HOME_VISIT.getMalariaPrevention())
+                .withHelper(malariaPreventionHelper)
+                .build();
+        actionList.put(context.getString(R.string.pnc_malaria_prevention), action);
+    }
+
+    private void evaluateCounselling() throws Exception {
+        HomeVisitActionHelper counsellingHelper = new HomeVisitActionHelper() {
+            private String couselling_child;
+
+            @Override
+            public void onPayloadReceived(String jsonPayload) {
+                try {
+                    JSONObject jsonObject = new JSONObject(jsonPayload);
+                    couselling_child = org.smartregister.chw.util.JsonFormUtils.getCheckBoxValue(jsonObject, "pnc_counselling");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.chw.util.JsonFormUtils.getCheckBoxValue' due to existing import 'org.smartregister.chw.util.JsonFormUtils'](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426631)